process/smartContract/process.go

```diff
@@ -576,6 +577,10 @@ func (sc *scProcessor) computeTotalConsumedFeeAndDevRwd(
 			}
 			consumedGas, err = core.SafeSubUint64(consumedGas, sentGas)
 			log.LogIfError(err, "computeTotalConsumedFeeAndDevRwd", "outTransfer.GasLimit")
+			consumedGas, err = core.SafeAddUint64(consumedGas, outAcc.GasUsed)
+			log.LogIfError(err, "computeTotalConsumedFeeAndDevRwd", "outAcc.GasUsed")
```

Comment:
Both log errors L581 and L583 display the same things

Reply:
done